anubis-management-api/anubis/policies/routers.py

```diff
@@ -508,6 +509,23 @@ def create_policy(
                 urllib.parse.quote(
                     policy.access_to)))
         logging.error(e)
+    opa_url = os.environ.get('OPA_ENDPOINT', 'http://127.0.0.1:8181')
```

Comment:
code below should be executed only if OPA_ENDPOINT is specified (leaving a way to use pull mode)